2023-07-14-sketch_map_tool_new_version.md

```diff
@@ -0,0 +1,86 @@
+
+The development of the first prototypes of the Sketch Map Tool started over four years ago in the [Waterproofing data project](https://www.geog.uni-heidelberg.de/gis/waterproofing.html) to support the collection of flood risk perception in two vulnerable Brazilian communities.
+
+<figure>
+<img src="https://www.missingmaps.org/img/images/missingmaps-blog-20230714_group.png">
+</figure>
+
+Since then, the tool has been tested in different contexts and projects around the world including Germany, Mozambique and Madagascar. In Mozambique, for example, users recorded flood areas and regions with increased crime using the Sketch Map Tool in workshops. The collected data was used in disaster risk reduction projects by the German Red Cross and [Mozambique Red Cross Society](https://piroi.croix-rouge.fr/piroi/members-of-piroi/mozambique-red-cross-society/?lang=en).
+
+<figure>
+<img src="https://www.missingmaps.org/img/images/missingmaps-blog-20230714_double.png">
+</figure>
+
+Currently, our former HeiGIT intern Kimon Letzner explores the potential and limitations of the Sketch Map Tool in the scope of his Master’s studies at Lund University. His focus is on the community mapping process in the context of the [Enhanced Vulnerability and Capacity Assessment (EVCA)](https://communityengagementhub.org/resource/evca-guide/) in the International Red Cross Red Crescent Movement based on a project with the [Colombian Red Cross National Society](https://www.cruzrojacolombiana.org/).
+
+<figure>
+<img src="https://www.missingmaps.org/img/images/missingmaps-blog-20230714_community_mapping.jpeg">
+<p class="caption">Community mapping (Letzner 2023)</p>
+</figure>
+
+In May 2023, the Sketch Map tool was presented and used in a practical exercise at a regional training of the [International Federation of Red Cross and Red Crescent Societies](https://www.ifrc.org/) “Road Map to Community Resilience via Enhanced Vulnerability and Capacity Assessment (EVCA)” in Kuala Lumpur, Malaysia. A write-up of the training can be found here. This event represented another crucial step towards implementing the tool in the standard processes of the Red Cross Red Crescent Movement worldwide.
+
+## Outlook
+
+The application examples presented here represent only a small part of the current projects in which the Sketch Map Tool has already been deployed. Due to the Sketch Map Tool’s functions and the easy handling of the tool, the HeiGIT team receives consistent information and ideas about applications and further development from the Red Cross and Red Crescent Movement and beyond. In cooperation with our partners, we are also currently conducting numerous workshops and training on the Sketch Map Tool and planning updates and expansions for the application.
+
+On Friday, June 9th at 4:30 PM EST, the Sketch Map Tool [was] presented at the State of the Map US 2023, the annual OSM conference in the USA, with additional information available [here](https://2023.stateofthemap.us/).
```

Comment:
is `[was]` supposed to be a link?

Reply:
No..the original referred to the future (will be presented) but since it already passed I changed the language